WBooks/Library/MainMenuController.swift

```diff
@@ -34,13 +34,31 @@ class MainMenuController: UIViewController {
         return false
     }
     
+    private func addBarButtons() {
+        let nav = self.navigationController?.navigationBar
+        //Title
+        self.title = "LIBRARY"
+        nav?.titleTextAttributes = [NSAttributedStringKey.foregroundColor: UIColor.white]
+        //Left notification button
+        let notificationsButton = UIButton(type: .system)
+        notificationsButton.setImage(#imageLiteral(resourceName: "ic_notifications").withRenderingMode(.alwaysOriginal), for: .normal)
+        notificationsButton.frame = CGRect(x: 0, y: 0, width: 34, height: 34)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: notificationsButton)
+        //Right search button
+        let searchButton = UIButton(type: .system)
+        searchButton.setImage(#imageLiteral(resourceName: "ic_search.png").withRenderingMode(.alwaysOriginal), for: .normal)
+        searchButton.frame = CGRect(x: 0, y: 0, width: 34, height: 34)
+        navigationItem.rightBarButtonItem = UIBarButtonItem(customView: searchButton)
```

Comment:
This is good, but it would be nice if you have an extension of `UIBarButtonItem` and define this bar buttons there, so if you need them for other views you can reuse the code.
If you need some help with this, let me know and we can have a call